client/run.go

```diff
@@ -254,6 +258,15 @@ func (cli *HyperClient) JsonFromCmdline(cmdArgs, cmdEnvs, cmdPortmaps []string,
 		}
 	}
 
+	for _, v := range cmdVols {
+		vol, volRef, err := parseVolume(v)
+		if err != nil {
+			return "", err
+		}
+		volumes = append(volumes, *vol)
+		volumesRef = append(volumesRef, *volRef)
```

Comment:
There is already one volStr argument. User will be confused if add more volName, right?
. The vol Name will be aded 5 random number in routine parseVolume, which is enough to make the name uniq.